### PR TITLE
docs: Go and REST Controller Binding docs

### PR DESCRIPTION
**Pull request by rolsonquadras**
Signed-off-by: Rolson Quadras <rolson.quadras@securekey.com>

### CODE REVIEW

File: docs/rest/README.md [hunk 1]
```diff
@@ -1,3 +1,22 @@
...
+The project can be used as a server agent, so that the implementors can just concentrate on writing the business logic.
+
+## Steps
+Well, running an agent in docker is as simple as [Build](agent_docker.md), [Run](agent_docker.md) and [Verify](openapi_demo.md). Also, agent can be run as a [binary](agent_cli.md). 
+
+## Integration
+The framework provides convenient Admin APIs to control the agent through [webhooks](agent_webhook.md). The agent triggers an event on registered webhooks. Refer [OpenAPI demo](openapi_demo.md) for detailed instructions.
+
+## References
+[Buind and Start Reference Agent using docker](agent_docker.md)
```
Comment (troyronda) [inline docs/rest/README.md]: Typo

File: docs/rest/README.md [hunk 2]
```diff
@@ -1,3 +1,22 @@
...
+
+[Run OpenAPI Demo](openapi_demo.md)
+
+[Agent webhook support](agent_webhook.md)
+
+[Buind and Start Reference Agent as a bin](agent_cli.md)
```
Comment (troyronda) [inline docs/rest/README.md]: typo

File: docs/rest/README.md (hunk 1 above)
Comment (rolsonquadras) [reply docs/rest/README.md]: [Done](https://github.com/hyperledger/aries-framework-go/compare/bc6930cd3baaf9434e5a0b6c93eff9ccddfeab40..0f81218001bfede4a6d557d06ebae59d552475dd#diff-f08a0875eeb20bae42697de5aa6b84edR12).

File: docs/rest/README.md (hunk 2 above)
Comment (rolsonquadras) [reply docs/rest/README.md]: [Fixed](https://github.com/hyperledger/aries-framework-go/compare/bc6930cd3baaf9434e5a0b6c93eff9ccddfeab40..0f81218001bfede4a6d557d06ebae59d552475dd#diff-f08a0875eeb20bae42697de5aa6b84edR18).

File: docs/test/bdd_instructions.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Instruction to run bdd tests
+# Run bdd tests
```
Comment (troyronda) [inline docs/test/bdd_instructions.md]: Why is this file still with the suffix _instructions? (I saw the others got renamed.)

Comment (rolsonquadras) [reply docs/test/bdd_instructions.md]: I missed the `test` folder. [Updated](https://github.com/hyperledger/aries-framework-go/compare/0f81218001bfede4a6d557d06ebae59d552475dd..9ce309499685bbe469cc977ba0746afd57600b88#diff-04c6e90faac2675aa89e2176d2eec7d8R48) the file names now.

**Pull request merged**